provision-contest/ansible/admin.yml

```diff
@@ -151,3 +137,33 @@
         etype: user
         permissions: rwx
         state: present
+
+    - name: Download offline repos
+      when: WF_RESTRICTED_NETWORK
+      become: true
+      become_user: domjudge
+      block:
+        - name: Add ccsadmin remote
+          git_config:
+            name: "remote.{{ item.remote }}.url"
+            value: "{{ item.url }}"
+            scope: local
+            repo: /home/domjudge/domjudge-checkout
+          loop:
+            - remote: origin
+              url: ansible@10.3.3.224:domjudge
+            - remote: github
+              url: git@github.com:DOMjudge/domjudge.git
+            - remote: sysops
+              url: domjudge@packages:domjudge.git
+            - remote: initial
+              url: ansible@domjudge-ccsadmin1:domjudge.git
```

Comment:
What's the difference with origin?

Reply:
One gets switched if the other git hosts get online, the other stays stable.